backend/app/Http/Controllers/Api/V1/DeviceIngestController.php

```
@@ -0,0 +1,80 @@
+        // than blindly acknowledging (a device must not stop retrying when its
+        // count was never applied).
+        ProcessMqttMessageJob::dispatchSync(
+            $connection->id,
+            DeviceEnrollmentController::PULSE_TOPIC,
+            $payload,
+            now()->toIso8601String(),
+        );
+
+        // Inspect the message the job just logged. A processing error means the
+        // count-step action failed — surface it so the sensor keeps retrying.
+        $message = MachineMessage::where('machine_connection_id', $connection->id)
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\MachineMessage::where().


(staticMethod.notFound)